doc/stdenv/meta.chapter.md

```diff
@@ -86,6 +86,17 @@ meta.platforms = lib.platforms.linux;
 
 Attribute Set `lib.platforms` defines [various common lists](https://github.com/NixOS/nixpkgs/blob/master/lib/systems/doubles.nix) of platforms types.
 
+### `badPlatforms` {#var-meta-badPlatforms}
+
+The list of Nix platform types on which the package is known not to be buildable.  Hydra will never create prebuilt binaries for these platform types, even if they are in `meta.platforms`.  In general it is preferable to set `meta.platforms = lib.platforms.all` and then exclude any platforms on which the package is known not to build.  For example, a package which requires dynamic linking and cannot be linked statically could use this:
+
+```nix
+meta.platforms = lib.platforms.all;
+meta.badPlatforms = [ lib.systems.inspect.patterns.isStatic ];
+```
+
+The `lib.meta.availableOn` function can be used to test whether or not a package is available (i.e. buildable) on a given platform.  Some packages use this to automatically detect the maximum set of features with which they can be built.  For example, `systemd` requires dynamic linking, and has a `meta.badPlatforms` setting similar to the one above.  Packages which can be built with or without `systemd` support will use `lib.meta.availableOn` to detect whether or not `systemd` is available on the `hostPlatform` for which they are being built; if it is not available (e.g. due to a statically-linked host platform like `pkgsStatic`) this support will be disabled by default.
```

Comment:
Please link to `lib.meta.availableOn`. If it's only documented in the source, provide a permalink to the current tip of `master`. Same for the `systemd` examples: just link to what you're referring to, then the reader can inspect the concrete cases. Also link to documentation for or definition of `hostPlatform` and `pkgsStatic`.

Reply:
d9e11f91d6b7